### DISCUSSION TITLE
Are the number of observations consistent across all model runs?

### DISCUSSION THREAD

**Discussion by esheehan-gsl** [NOAA-GSL/unified-graphics#142] [Development]
@terraladwig I have a couple of questions about the observations in the diag files. I want to make sure I’m not making wrong assumptions, because it has a significant impact on how we store the data.

**First, will every diag file have exactly the same number of observations?** My guess is no, the number of observations can change from hour to hour.

**Second, will there be only one observation per `Station_ID`, or can a station have multiple observations?** I think I recall being told that there can be multiple observations per station, because some stations take readings more than once an hour, but I want to be sure.

**Comment by terraladwig** [comment]
1. Correct, the number of obs will vary.  Sometimes an instrument is broken or there are more/less aircraft data at a time, etc. 
2. Correct, there can be more than one observation per station.  Like you said, some stations produce multiple obs per hour.  Also, depending on how you are sorting, there are multiple types at a station.  For example, a temperature, dew point, and wind at the same station.  Another note, I'm not sure if all observations have a Station_ID.  The surface data should have that, but other platforms that we will be using may not have it.  (e.g. maybe aircraft has a tail # and no station ID)
